services/src/main/scala/cromwell/services/metadata/WorkflowQueryKey.scala

```diff
@@ -37,16 +40,32 @@ object WorkflowQueryKey {
     override val name = "Name"
 
     override def validate(grouped: Map[String, Seq[(String, String)]]): ErrorOr[List[String]] = {
-
       val values = valuesFromMap(grouped).toList
-      val nels = values map {
+      val nels:List[data.ValidatedNel[String,String]] = values map {
```

Comment:
(Check out the diff under the "Files Changed". The aforementioned magic in `.idea/inspectionProfiles/Project_Default.xml` is explicitly removed in this PR. Not sure why?)